### PR TITLE
SDK-4164: Custom Templates

### PR DESCRIPTION
**Pull request by akashvercetti**

### CODE REVIEW

File: Samples/Cordova/ExampleProject/platforms/android/android.json [hunk 1]
```diff
@@ -63,9 +63,9 @@
   },
   "installed_plugins": {
     "clevertap-cordova": {
-      "CLEVERTAP_ACCOUNT_ID": "67Z-RRK-696Z",
-      "CLEVERTAP_TOKEN": "322-1a6",
-      "CLEVERTAP_REGION": "sk1-staging-25",
+      "CLEVERTAP_ACCOUNT_ID": "TEST-4RW-944-876Z",
```
Comment (darshanclevertap) [inline Samples/Cordova/ExampleProject/platforms/android/android.json:66]: We should not push any credentials to master

Comment (akashvercetti) [reply Samples/Cordova/ExampleProject/platforms/android/android.json:66]: @Anush-Shand could you reverse this?

Comment (darshanclevertap) [reply Samples/Cordova/ExampleProject/platforms/android/android.json:66]: I will approve this PR, just ensure we don't push any CT credentials to master in the final PR

**Review by darshanclevertap — APPROVED** [review]
@Anush-Shand please remove CT Credentials from Samples. Approving this PR to unblock iOS

**Pull request merged**